app/role_data.go

```diff
@@ -103,11 +104,13 @@ func addRoles(management *config.ManagementContext) (string, error) {
 
 	rb.addRoleTemplate("Manage Nodes", "nodes-manage", "cluster", true, false, false).
 		addRule().apiGroups("management.cattle.io").resources("nodes", "nodepools").verbs("*").
-		addRule().apiGroups("*").resources("nodes").verbs("*")
+		addRule().apiGroups("*").resources("nodes").verbs("*").
+		addRule().apiGroups("management.cattle.io").resources("clustermonitorgraphs").verbs("get", "list", "watch")
```

Comment:
The clustermonitorgraphs resource includes graphs like `apiserver-request-count` which is outside of `node-management` scope. I think we should narrow down the pre-canned rule, so that we only grant access to `node-cpu-usage, node-disk-io, etc.` clustermonitorgraphs for `node-manage` role.
Can you try if this helps:
```
nodeMonitorGraphs := []string{"node-cpu-usage",...}
addRule().apiGroups("management.cattle.io").resources("clustermonitorgraphs").resourceNames(nodeMonitorGraphs).verbs("get", "list", "watch")
```
Similarly, check `nodes-view, workloads-manage, workloads-view` roles.

Reply:
Hi, the ```node view``` and ```node manage``` role can still view the cluster view, so I add them all.